ui/public/images/icons/console.svg

```diff
@@ -0,0 +1,45 @@
+    </defs>
+    <g id="Symbols" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="exec-/-button-/-default" transform="translate(-9.000000, -7.000000)">
+            <g>
+                <g id="🎨-Design-guidelines-/-❏-Elevations-/-$box-shadow-low">
+                    <g id="Low">
+                        <use fill="black" fill-opacity="1" filter="url(#filter-2)" xlink:href="#path-1"></use>
+                        <use fill="#FFFFFF" fill-rule="evenodd" xlink:href="#path-1"></use>
+                    </g>
+                </g>
+                <mask id="mask-4" fill="white">
+                    <use xlink:href="#path-3"></use>
+                </mask>
+                <rect stroke="#BBC4D1" x="0.5" y="0.5" width="71" height="28" rx="2"></rect>
+                <g id="icons-/-console" mask="url(#mask-4)">
+                    <g transform="translate(8.000000, 6.500000)">
+                        <mask id="mask-6" fill="white">
+                            <use xlink:href="#path-5"></use>
+                        </mask>
+                        <use id="Combined-Shape" stroke="none" fill="#000000" fill-rule="nonzero" xlink:href="#path-5"></use>
+                        <g id="🎟-Color-/-UI-Gray-/-$ui-gray-800" stroke="none" fill="none" mask="url(#mask-6)">
+                            <rect fill="#373A42" x="0" y="0" width="16" height="16"></rect>
+                            <rect id="color" fill="#373A42" fill-rule="evenodd" x="0" y="0" width="16" height="16"></rect>
+                        </g>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
Ember Inline SVG will do this at build time, but I think this should be run through [svgo](https://github.com/svg/svgo) and saved that way. Just be mindful of preserving the `viewBox`.

Reply:
Makes sense, done, thanks!